### PR TITLE
Create 2021-74.md

### PR DESCRIPTION
**Pull request by billfox1**

### CODE REVIEW

**Review by ninetreats — DISMISSED** [review]
Looks good to me.

File: release-notes/2021-74.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        p.acceptedSecurityPolicyOn as "Accepted Security Policy", 
+        p.backgroundCheck as "Completed Background Check"
+      (edited)
+
+
+## Integrations
+
+### Auth0
+
+- Initial release of an integration with Auth0. Please see [documentation](https://github.com/JupiterOne/graph-auth0/blob/master/docs/jupiterone.md)
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:34]: JupiterOne now supports Auth0 with an initial integration. please see [documentation](https://github.com/JupiterOne/graph-auth0/blob/master/docs/jupiterone.md for more details.

File: release-notes/2021-74.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+for more details.
+
+### Google Cloud
+
+- [#255](https://github.com/JupiterOne/graph-google-cloud/issues/255) - Ensure
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:39]: Now `google_redis_instance` and `google_memcache_instance` use unique `_key` values

File: release-notes/2021-74.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  that `google_redis_instance` and `google_memcache_instance` use unique `_key`
+  values
+
+- [#243](https://github.com/JupiterOne/graph-google-cloud/issues/243) - Always
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:43]: Now JupiterOne assigns boolean value to `public` property on `google_storage_bucket`

File: release-notes/2021-74.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  assign boolean value to `public` property on `google_storage_bucket`
+
+- [#248](https://github.com/JupiterOne/graph-google-cloud/issues/248) - Always
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:46]: JupiterOne now always execute steps in org child projects even if the child project has service disabled.

File: release-notes/2021-74.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+- Added support for ingesting the following **new** relationships:
+
+  | Source                   | \_class    | Target                   |
+  | ------------------------ | ---------- | ------------------------ |
+  | `google_compute_network` | `CONNECTS` | `google_compute_network` |
+
+### GitHub
+
+- [JupiterOne/integrations#5](https://github.com/JupiterOne/integrations/issues/5)
+  Use `name || login` for `displayName` of `Account` and `User` entities.
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:65]: JupiterOne now uses `name || login` for `displayName` of `Account` and `User` entities.

File: release-notes/2021-74.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+### Qualys
+
+- Configure host detection types with `Potential,Confirmed` in the
+  `vmdrFindingTypes` instance config field (`VMDR_FINDING_TYPES` in `.env`).
+
+- Only host detections with types `Potential,Confirmed` are ingested by default.
```
Comment (AnneNielsenJ1) [inline release-notes/2021-74.md:72]: JupiterOne by default ingests host detections with types `Potential, Confirmed`. We recommend avoiding ingesting `Info` detections until you are ready to process them in a meaningful way.

**Review by AnneNielsenJ1 — CHANGES_REQUESTED** [review]
@billfox1 I made some suggested tweaks on wording to standardize our tone a bit.

**Review by AnneNielsenJ1 — APPROVED** [review]
Just one typo to address, otherwise looks great. Thank you!

**Review by AnneNielsenJ1 — APPROVED** [review]

**Review by AnneNielsenJ1 — APPROVED** [review]

File: release-notes/2021-74.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
+# JupiterOne 2021.74 Release
+
+2021-06-30
+
+## New Features and Improvements
+
+- JupiterOne now provides an updated Compliance experience including a new total compliance status which aggregates adherence to selected individual compliance standards into a snapshot view as well as highlighting open risks, controls, people, and vendors.
+
+- “Approves” asset relationships will now appear as green in the graph and “Denies” relationships will now appear as red.
```
Comment (erkangz) [inline release-notes/2021-74.md:9]: @billfox1 @AnneNielsenJ1 this should be Allows, not Approves

Comment (billfox1) [reply release-notes/2021-74.md:9]: Resolved in https://github.com/JupiterOne/docs/pull/270

**Pull request merged**